test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/MathTest.scala

```diff
@@ -81,23 +81,37 @@ class MathTest {
   }
 
   @Test def cbrt(): Unit = {
-    assertTrue(1 / Math.cbrt(-0.0) < 0)
+    assertSameDouble(-0.0, Math.cbrt(-0.0))
+    assertSameDouble(0.0, Math.cbrt(0.0))
     assertEquals(3.0, Math.cbrt(27.0), 0.0)
     assertEquals(100.0, Math.cbrt(1000000.0), 0.0)
     assertEquals(1000.0, Math.cbrt(1000000000.0), 0.0)
     assertEquals(-100000000.0, Math.cbrt(-1.0E24), 0.0)
     assertEquals(-4039.0E8, Math.cbrt(-65890311319.0E24), 0.0)
+    assertTrue(Math.cbrt(Double.NaN).isNaN)
+    assertSameDouble(Double.PositiveInfinity, Math.cbrt(Double.PositiveInfinity))
+    assertSameDouble(Double.NegativeInfinity, Math.cbrt(Double.NegativeInfinity))
   }
 
   @Test def log1p(): Unit = {
     assertTrue(Math.log1p(-2.0).isNaN)
     assertTrue(Math.log1p(Double.NaN).isNaN)
     assertEquals(0.0, Math.log1p(0.0), 0.0)
```

Comment:
You should change this test to use `assertSameDouble`. That would expose the big I commented about above.